src/SFA.DAS.EmployerFinance.Web/Filters/LevyEmployerTypeOnly.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Web.Mvc;
+using System.Web.Routing;
+using SFA.DAS.Common.Domain.Types;
+using SFA.DAS.EAS.Account.Api.Client;
+using SFA.DAS.EAS.Account.Api.Types;
+using SFA.DAS.EmployerFinance.Web.Helpers;
+
+namespace SFA.DAS.EmployerFinance.Web.Filters
+{
+    public class LevyEmployerTypeOnly : ActionFilterAttribute
+    {
+        public override void OnActionExecuting(ActionExecutingContext filterContext)
+        {
+            try
+            {
+                var hashedAccountId = filterContext.ActionParameters["HashedAccountId"].ToString();
```

Comment:
Would it be better to check for HashedAccountId explicitly rather than rely on the try, catch?